### PR TITLE
B thomas22tech patch 1

### PR DESCRIPTION
**Pull request by BThomas22tech**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6741"></a>

### CODE REVIEW

File: data/BThomas22tech.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Brandon Thomas",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Full Stack Python developer looking to change the world!"
+  "avatar": "https:github.com/BThomas22tech.png",
```
Comment (amandamartin-dev) [inline data/BThomas22tech.json:6]:
```suggestion

```
no longer needed, please remove

File: data/BThomas22tech.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Brandon Thomas",
+  "type": "personal",
+  "displayStatsPublic": true,
```
Comment (amandamartin-dev) [inline data/BThomas22tech.json:4]:
```suggestion
 
```
no longer needed, please remove

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

File: data/BThomas22tech.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Brandon4Tenn",
+      "icon": "FaTwitter"
+},
+{
+      "name": "Follow me on Github",
+      "url": "https://github.com/BThomas22tech",
+      "icon": "FaGithub"
+},
```
Comment (HassanTanveer) [inline data/BThomas22tech.json:16]:
```suggestion
}
```
Please remove the trailing comma

**Review by HassanTanveer — CHANGES_REQUESTED** [review]

File: data/BThomas22tech.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Brandon Thomas",
+  "type": "personal",
+ 
+  "bio": "Full Stack Python developer looking to change the world!"
```
Comment (SaraJaoude) [inline data/BThomas22tech.json:5]: needs a comma:
```suggestion
  "bio": "Full Stack Python developer looking to change the world!",
```

**Review by amandamartin-dev — APPROVED** [review]

**Pull request merged**